das/daser.go

```diff
@@ -73,7 +100,16 @@ func NewDASer(
 		subscriber:     newSubscriber(),
 		subscriberDone: make(chan struct{}),
 	}
-	d.sampler = newSamplingCoordinator(concurrencyLimit, samplingRange, getter, d.sample)
+
+	for _, applyOpt := range options {
+		applyOpt(d)
+	}
+
+	if len(options) == 0 {
+		log.Warn("das/daser: no options applied, daser parameters will be instantiated with zero-values, make sure to configure the daser")
```

Comment:
I wonder if user could be scared by this. Like we are pushing him to do some proper set up here, but does he need to? I think it should work fine with just default config.